tiledb/sm/rtree/rtree.cc

```diff
@@ -206,6 +206,11 @@ double RTree::range_overlap(const std::vector<const T*>& range, const T* mbr) {
         mbr_range = std::nextafter(mbr_range, max);
     }
     ratio *= (double)overlap_range / mbr_range;
+
+    // If ratio goes to 0, then this is a point query. We need to reset ratio
```

Comment:
The ratio could go down to 0 even if the subarray query is not a point query, but still extremely small. So I would just rephrase and say that "If ration goes to 0, then the subarray overlap is much smaller than the volume of the MBR. Since we have already guaranteed that there is an overlap above, we should set the ratio to epsilon." Then, I would also make it a bit more explicit by setting `ratio = std::nextafter(0, max);` just for clarity.

Reply:
👍